packages/schema/src/interfaces.ts

```diff
@@ -29,6 +29,10 @@ import { IRasterSource } from './_interface/rastersource';
 
 export { IGeoJSONSource } from './_interface/geojsonsource';
 
+export type CenterPosition = {
```

Comment:
Nice! Hope you don't mind me leaving a comment on your draft :) 

Nit: I feel this name would work better if not for the `zoom` attribute. What about `ViewportPosition` or `ViewportState`?